src/models/galp6/repairs.md

```diff
@@ -0,0 +1,227 @@
+5. Re-connect the CMOS battery and main battery, and replace the M.2 drive and bottom panel.
+6. Power up the machine. The system may power itself off and on after initial boot; this is normal behavior when the CMOS has been reset.
+
+## Replacing the speakers:
+
+The system has two bottom-firing speakers, which can be removed and replaced individually.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 15 minutes  
+**Difficulty:** Easy <span style="color:green;">●</span>
+
+### Steps to replace the speakers:
+
+1. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover). For the right speaker, also [remove the battery](#removing-the-battery) and [remove the SSD](#replacing-an-m2nvme-ssd).
+2. Unscrew the speaker screws.
+    - Each speaker has two screws, which are set inside of rubber rings to reduce vibration.
+
+![Speaker screws and connectors](./img/speakers.webp)
+
+3. Disconnect the speaker wire from the motherboard and lift the speaker out of the machine.
+    - The speaker wire may be partially routed behind other components or tape; be careful when removing it.
+4. Place the new speaker in the machine, plug in the connector, replace the screws, and reinstall the SSD and battery (if necessary) and bottom panel.
+
+## Replacing the cooling system:
+
+The Galago Pro 6 has a single heatsink with two fans for the CPU and integrated GPU. Both fans are separate from the heatsink and can be replaced independently.
+
+If a fan becomes noisy and cleaning it out doesn't fix the issue, you may need a new fan. [Contact support](https://support.system76.com) to start a warranty claim or parts purchase.
+
+Depending on your climate and the age of the machine, it may be necessary to apply new thermal paste between the CPU and the heatsink. Thermal paste helps facilitate effective heat transfer between the CPU and the cooling equipment. These instructions can also be used in the unlikely event that your heatsink needs to be replaced.
+
+**Tools required:** Cross-head (Phillips) screwdriver, thermal paste  
+**Time estimate:** 20 minutes  
+**Difficulty:** High <span style="color:red;">●</span>  
+
+### Steps to replace the fans:
+
+1. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover).
+2. Remove any clear tape that is securing the fan wires.
+3. Unplug the white fan connector (highlighted  below) for each fan you are removing from the system.
+
+![Fan screws](./img/fan-screws.webp)
+
+4. Remove the two fan screws (highlighted green above) for each fan you are removing.
+5. Replace the fans, screw in the two fan screws for each fan, and plug the white fan connectors into the system.
+
+### Steps to replace the heatsink/thermal paste:
+
+1. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover) and [remove the fan closest to the CPU](steps-to-replace-the-fans).
```

Comment:
The fan removal link 404's

Reply:
Fixed with [86912cc](https://github.com/system76/tech-docs/pull/142/commits/86912cce030b3754cf51cbccf3709f63a2919c92).